### DISCUSSION TITLE
RESP3 support

### DISCUSSION THREAD

**Discussion by 93578237** [dsoftwareinc/fakeredis-py#352] [Q&A]
Hi, to use [client-side caching](https://redis.io/docs/latest/develop/clients/client-side-caching/), resp3 should be used, but fakeredis is failing with 
```
redis.exceptions.ResponseError: unknown command `hello`, with args beginning with:
```

I see that hello command is not implemented https://github.com/cunla/fakeredis-py/blob/0ad44e874026a09cf1962889c4ec12f64551cab4/docs/supported-commands/Redis/CONNECTION.md?plain=1#L95

but how much work is required to support resp3?

**Comment by cunla** [comment]
Implementing `hello` command is one thing - That should be relatively easy (the `proto` for the client can be "ignored").

Support for RESP3 is another thing - that would be more challenging.
Let me see whether I can design it. Would you like to participate in implementing this?

**Reply by 93578237** [reply]
I am not sure where to start. I have not seen yet how fakeredis works.
